rpc/client.go

```diff
@@ -157,6 +157,19 @@ type ClientInformational interface {
 
 	// GetChainspec returns the raw bytes of the chainspec.toml, accounts.toml and global_state.toml files as read at node startup.
 	GetChainspec(ctx context.Context) (InfoGetChainspecResult, error)
+
+	// GetValidatorRewardByEraID returns the reward for a given era and a validator
+	GetValidatorRewardByEraID(ctx context.Context, validator keypair.PublicKey, eraID uint32) (InfoGetRewardResponse, error)
```

Comment:
I noticed `eraID` is some times `uint64`, and some times `uint32`. According to the schema, `uint64` is the correct type.